source/_integrations/waze_travel_time.markdown

```diff
@@ -62,6 +62,62 @@ template:
 ```
 
 {% endraw %}
+Automations / Scrips can be setup to poll Waze on any changes to the input_select.
```

Comment:
Let's document stuff like this generically, and not on integration level

(as this is not integration specific).

Reply:
Sure, but the issue at hand is: We don't have centralized documentation on the matter. Which is my suggestion to do, instead of what is added here.